src/state/image-dataset/csv-dataset/test/CsvRequest.test.ts

```diff
@@ -0,0 +1,39 @@
+import { expect } from "chai";
```

Comment:
Unit tests are purposefully very basic right now! Will be adding more robust checks in a future PR.